chainweb.cabal

```diff
@@ -100,8 +100,6 @@ library
         , Chainweb.BlockHeader
         , Chainweb.BlockHeader.Genesis
         , Chainweb.BlockHeader.Genesis.DevelopmentPayload
-        , Chainweb.BlockHeader.Genesis.Testnet00Payload
-        , Chainweb.BlockHeader.Genesis.Testnet01Payload
         , Chainweb.BlockHeader.Genesis.Testnet02Payload
```

Comment:
These module names are generated by `ea` and are a direct textual rendering from the `ChainwebVersion` value.

Reply:
Hmmm, my concern here would be that now the process of generating a new set of genesis blocks starts to become an endeavor that requires expertise and that can easily be gotten wrong.  Not sure how to avoid that.